js/AddTodolistItem.js

```diff
@@ -0,0 +1,155 @@
+import {
+    getUserIdAndDeleteTodolist,
+    getUserIdAndItemComplete,
+    getUserIdAndModifyTodolist,
+    getUserIdAndAddTodolist
+} from './Server.js';
+
+function addTodolistItem(value, item) {
+    checkServerItemComplete(value,item);
+    clickEraseButton();
+    clickCheckboxButton();
+    clickLabel();
+    //strong태그 안에 있는 count 값 적용시키기
+    document.querySelector('.todo-count > strong').innerText = document.querySelector('.todo-list').childElementCount;
+
+}
+function clickSet() {
+//전체보기 버튼 클릭시 모든 내용 display
+    document.querySelector('.all').addEventListener('click', onAllFilterHandler);
+
+//해야할 일 버튼 클릭시 class가 completed인 것들은 display가 none이 되도록
+    document.querySelector('.active').addEventListener('click', onActiveFilterHandler);
+
+//완료한 일 클릭시 class가 active인 것들은 display가 none이 되도록
+    document.querySelector('.completed').addEventListener('click', onCompletedFilterHandler);
+}
+
+function onAllFilterHandler(){
+    document.querySelectorAll('.todo-list > li').forEach(x => x.style.display = '');
```

Comment:
forEach 는 실질적으로 반환값을 사용하지 않으니 중괄호로 감싸주면 좋을 것 같아요
```suggestion
    document.querySelectorAll('.todo-list > li').forEach(x => { x.style.display = '' } );
```